webanno-io-tsv/src/main/java/de/tudarmstadt/ukp/clarin/webanno/tsv/WebannoTsv1Reader.java

```diff
@@ -146,7 +146,7 @@ private void createDependency(JCas aJCas, Map<Integer, String> tokens,
                 // if span A has (start,end)= (20, 26) and B has (start,end)= (30, 36)
                 // arc drawn from A to B, dependency will have (start, end) = (20, 36)
                 // arc drawn from B to A, still dependency will have (start, end) = (20, 36)
-                int begin = 0, end = 0;
+                int begin, end;
```

Comment:
I prefer having explicit initalizers for primitive variables, even if they initialize to the same value that the JVM would use.